src/middlewares/userValidationMiddleware.ts

```diff
@@ -0,0 +1,20 @@
+import Joi from 'joi';
+import { Request, Response, NextFunction } from 'express';
+
+const UserDataSchema = Joi.object({
+    login: Joi.string().required(),
+    password: Joi.string()
+        .pattern(new RegExp(/^[0-9]*[a-zA-Z]+[a-zA-Z0-9]*$/))
+        .required(),
+    age: Joi.number().integer().min(4).max(130).required()
+});
+
+export const userValidator = (req: Request, res: Response, next: NextFunction) => {
+    const { error } = UserDataSchema.validate(req.body);
+
+    if (error) {
+        return res.status(400).json({ status: 400, message: 'Validation Error' });
```

Comment:
it's possible to take error message from Joi `error.details` instead of just 'Validation Error'